spec/helpers/devise_helper_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DeviseHelper do
```

Comment:
Where is this helper used? I vaguely remember changes to Devise that may have made this obsolete... but I may be confusing with something else.

Reply:
It's used for these pages https://github.com/LucasAnderson1984/imas/tree/master/app/views/users

Devise did deprecate the method `devise_error_messages!`, but I haven't seen how or if they replaced it.